content/docs/import/import-from-postgres.md

```diff
@@ -149,13 +149,13 @@ The `pg_dump` and `pg_restore` commands provide numerous advanced options, some
 - `-Z`: Defines the compression level to use when using a compressible format. 0 means no compression, while 9 means maximum compression. In general, we recommend a setting of 1. A higher compression level slows the dump and restore process but also uses less disk space.
 - `--lock-wait-timeout=20s`: Error out early in the dump process instead of waiting for an unknown amount of time if there is lock contention.
 Do not wait forever to acquire shared table locks at the beginning of the dump. Instead fail if unable to lock a table within the specified timeout.`
-- `-j <njobs>`: Consider this option for medium (>1 GB) and large databases (>5) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
+- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
```

Comment:
```suggestion
- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5 GiB) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
```

Reply:
btw, i did not realize we suggest this classification on medium and large!
do you remember, @danieltprice , where did we get these definitions from?

Reply:
FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

Reply:
@stepashka I added the database size recommendations with respect to when you use the -j option and psql vs pg_dump. I found this recommended elsewhere. You wouldn't need a parallel load for a small dataset, and avoiding piping for large datasets seems like good general advice. Let me know if you have any concerns.

Reply:
I've removed the specific size classifications.